src/commands/flinkStatements.ts

```diff
@@ -218,7 +201,7 @@ export async function submitFlinkStatementCommand(
 
   try {
     const restResponse = await submitFlinkStatement(submission);
-    const newStatement = restFlinkStatementToModel(restResponse);
+    const newStatement = restFlinkStatementToModel(restResponse, computePool);
```

Comment:
New 2nd arg for restFlinkStatementToModel, any `IProviderRegion` implementation describing the cloud provider / region that this statement result was loaded from. A `CCloudFlinkComputePool` suffices nicely.